locales/en/messages.json

```diff
@@ -2157,16 +2160,22 @@
     },
     "firmwareFlasherBaudRate": {
         "message": "Baud Rate"
-    },    
-    "firmwareFlasherShowDevelopmentReleases":{
-        "message": "Show unstable releases"
-    },
-    "firmwareFlasherShowDevelopmentReleasesDescription":{
-        "message": "Show Release-Candidates and Development Releases."
     },
     "firmwareFlasherOptionLoading": {
         "message": "Loading ..."
     },
+    "firmwareFlasherOptionLabelBuildTypeRelease": {
+        "message": "Release"
+    },
+    "firmwareFlasherOptionLabelBuildTypeReleaseCandidate": {
+        "message": "Release Candidate"
+    },
+    "firmwareFlasherOptionLabelBuildTypeDevelopment": {
+        "message": "Development"
+    },
+    "firmwareFlasherOptionLabelBuildTypeAKK": {
+        "message": "3.3 AKK & RDQ VTX Patch"
```

Comment:
This will only be used in 3.3? I think this must not contain version, the version goes into the other dropdown.

Reply:
Yes, it's only for 3.3, for 3.4 we have the fix built-in. Having no version listed here makes it a bit confusing for user.

Reply:
Ok, then it's ok.